q2_assembly/plugin_setup.py

```diff
@@ -179,11 +180,15 @@
     inputs={
         "contigs": SampleData[Contigs],
         "reads": SampleData[SequencesWithQuality | PairedEndSequencesWithQuality],
-        "references": List[FeatureData[Sequence]],
+        "references": GenomeData[DNASequence],
         "mapped_reads": SampleData[AlignmentMap],
     },
     parameters=quast_params,
-    outputs={"results_table": QUASTResults, "visualization": Visualization},
+    outputs=[
+        ("results_table", QUASTResults),
+        ("visualization", Visualization),
+        ("genomes", GenomeData[DNASequence]),
```

Comment:
Maybe we can call this output `reference_genomes` just to be a bit more explicit.